src/index.js

```diff
@@ -49,20 +51,71 @@ app.use(constants.GRAPHQL_PATH, (req, res, next) => {
     schema,
     context: {
       // Models
-      ProcedureModel,
+      Procedure,
```

Comment:
Hier wurde der Context name des Procedure Models geändert, ohne dass dies in den Revolvern beachtet wurde, dadurch sind diese so derzeit alle kaputt und der client würde keine Daten mehr bekommen